modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/action/DistributedActionProcessor.java

```diff
@@ -61,9 +61,9 @@
     public DistributedActionProcessor(GridKernalContext ctx) {
         super(ctx);
 
-        this.mgr = ((ManagementConsoleProcessor)ctx.managementConsole()).webSocketManager();
-        this.sesRegistry = ((ManagementConsoleProcessor)ctx.managementConsole()).sessionRegistry();
-        this.locNodeConsistentId = String.valueOf(ctx.grid().localNode().consistentId());
+        mgr = ((ManagementConsoleAgent)ctx.managementConsole()).webSocketManager();
```

Comment:
Why we need this cast. Can we add this method to interface?

Reply:
No, this is implementation details so far.